src/TimeSkipper/TimeSkipper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Lexide\Chronos\TimeSkipper;
+
+class TimeSkipper implements TimeSkipperInterface
+{
+
+    public function skip(float|int $seconds): void
+    {
+        sleep((int) $seconds);
```

Comment:
We don't need the explict cast to int. Can also remove the float hint from the parameter.

Reply:
My bad I noticed its from an interface.